src/service/action-impl.js

```diff
@@ -492,7 +496,8 @@ export class ActionService {
     let n = target;
     while (n) {
       const actionInfos = this.matchActionInfos_(n, actionEventType);
-      if (actionInfos) {
+      const isEnabled = !matches(n, DISABLED_PSEUDOCLASS_);
```

Comment:
@dvoytenko, `el.disabled` is indeed supported but it does not indicate if the element is **actually disabled** (a term from [specs](https://www.w3.org/TR/html5/disabled-elements.html#concept-element-disabled)). For that one can use `:disabled` or compute it themself by traversing the tree up.